GUI/GUI.gd

```diff
@@ -23,7 +24,7 @@ func _on_wave_changed(wave: Wave, wave_count: int, total_wave_count: int):
 	$WaveCounter.text = "WAVE %d/%d" % [wave_count, total_wave_count]
 	$ZombiesLeft.text = "%d ZOMBIES LEFT" % wave.total_count
 	
-func _on_enemy_died(died: int, total: int):
+func _on_enemy_died(enemy: Enemy, died: int, total: int):
```

Comment:
why got `enemy` here but unused?

Reply:
Because the signal emits the dead enemy, total death count, and total mob count, so regardless the function has to have these three.